c7n/resources/rds.py

```diff
@@ -1611,6 +1611,7 @@ class ModifyDb(BaseAction):
                         'Domain',
                         'CopyTagsToSnapshot',
                         'MonitoringInterval',
+                        'MonitoringRoleARN',
```

Comment:
I was thinking to switch them to use shape validation

Reply:
Hmm, I hadn't thought about that. We can bail on this one if you're putting something else together.

Reply:
I can also try the shape_validate approach if we think its the right way to go!

Reply:
Haven't started writing anything as such! Had just taken a look yesterday! Looks like there are a couple of new things added to the api. Thought it would be best to switch the validation to use shape validate

Reply:
Ok cool, I'll take a look at it. Are the shapes generally accurate? E.g. no discrepancies with API behavior?

Reply:
Not sure though if it'll be easy to do that keeping in mind the way this action is written. Would be nice to get your thoughts @kapilt

Reply:
the issue is compatibility one extant policy re switch.